app/tests/api/test_websocket.py

```
@@ -0,0 +1,24 @@
+    """
+    with client.websocket_connect("/api/v1/ws") as websocket:
+        websocket.send_text("Ping")
+        assert True
+
+def test_websocket_validates_dependency():
+    """
+    If we had real auth, we would test that invalid tokens fail here.
+    For now, since get_current_user_id always returns a user, 
+    we just verify the connection opens successfully.
+    """
+    with client.websocket_connect("/api/v1/ws") as websocket:
```

Ruff (0.14.14)

Local variable `websocket` is assigned to but never used

Remove assignment to unused variable `websocket`

(F841)